nbdev/showdoc.py

```diff
@@ -362,7 +362,8 @@ def show_doc(elt, doc_string:bool=True, name=None, title_level=None, disp=True,
     if doc_string and inspect.getdoc(elt):
         s = inspect.getdoc(elt)
         # show_doc is used by doc so should not rely on Config
-        try: monospace = (get_config().get('monospace_docstrings') == 'True')
+        # We use except because `get_config` raises Exception if not found
+        try: monospace = get_config().d.getboolean('monospace_docstrings', False)
         except: monospace = False
```

Comment:
Can you constrain the `except` clause to just the type we need to catch, or remove the `try/catch` if we don't need it any more?

Reply:
@jph00 we can safely remove the try/catch, since `get_config` will go ahead and make its own config if one isn't found, and we include a default for `getboolean` if it's not found